diff --git a/drv.c b/drv.c
index 9dd7471..7fa6093 100755
--- a/drv.c
+++ b/drv.c
@@ -43,6 +43,7 @@ MODULE_LICENSE("GPL");
 #define PCI_DEVICE_ID_AMD_19H_M70H_ROOT     0x14e8
 #define PCI_DEVICE_ID_AMD_MI200_ROOT        0x14bb
 #define PCI_DEVICE_ID_AMD_MI300_ROOT        0x14f8
+#define PCI_DEVICE_ID_AMD_VANGOGH_ROOT      0x1645
 
 #define MAX_ATTRS_LEN                      13
 
@@ -451,6 +452,7 @@ static struct pci_device_id ryzen_smu_id_table[] = {
     { PCI_DEVICE(PCI_VENDOR_ID_AMD, PCI_DEVICE_ID_AMD_19H_M70H_ROOT) },
     { PCI_DEVICE(PCI_VENDOR_ID_AMD, PCI_DEVICE_ID_AMD_MI200_ROOT) },
     { PCI_DEVICE(PCI_VENDOR_ID_AMD, PCI_DEVICE_ID_AMD_MI300_ROOT) },
+    { PCI_DEVICE(PCI_VENDOR_ID_AMD, PCI_DEVICE_ID_AMD_VANGOGH_ROOT) },
     { }
 };
 MODULE_DEVICE_TABLE(pci, ryzen_smu_id_table);
diff --git a/smu.c b/smu.c
index 3c439e6..cbfa69d 100755
--- a/smu.c
+++ b/smu.c
@@ -293,6 +293,7 @@ int smu_resolve_cpu_class(struct pci_dev *dev) {
       g_smu.codename = CODENAME_MATISSE;
       break;
     case 0x90:
+    case 0x91:
       g_smu.codename = CODENAME_VANGOGH;
       break;
     default:
@@ -416,13 +417,11 @@ int smu_init(struct pci_dev *dev) {
   case CODENAME_PHOENIX:
   case CODENAME_STRIXPOINT:
   case CODENAME_HAWKPOINT:
+  case CODENAME_VANGOGH:
     g_smu.addr_rsmu_mb_cmd = 0x3B10A20;
     g_smu.addr_rsmu_mb_rsp = 0x3B10A80;
     g_smu.addr_rsmu_mb_args = 0x3B10A88;
     goto LOG_RSMU;
-  case CODENAME_VANGOGH:
-    pr_debug("RSMU Mailbox: Not supported or unknown, disabling use.");
-    goto MP1_DETECT;
   default:
     pr_err("Unknown processor codename: %d", g_smu.codename);
     return -ENODEV;
